x-pack/plugins/fleet/common/openapi/paths/uninstall_tokens@{uninstall_token_id}.yaml

```diff
@@ -0,0 +1,39 @@
+get:
+  summary: Get one decrypted uninstall token by its ID
+  tags:
+    - Uninstall tokens
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              item:
+                type: object
+                properties:
+                  id:
+                    type: string
+                  token:
```

Comment:
is it expected that the `token` is returned in the response?

Reply:
yes, this route is intended to return the decrypted token itself